### PR TITLE
Add StyleCop checks for Game, Editor and Mod dlls to make.ps1

### PR DESCRIPTION
**Pull request by reaperrr**
Allows Windows users to check Game.exe and dlls more easily.

### CODE REVIEW

**Comment by Mailaender** [comment]
Good idea. I think you can also add OpenRA.Editor as https://github.com/OpenRA/OpenRA/pull/4496 will probably be blocked and delayed a lot longer.

**Comment by reaperrr** [comment]
Updated. Added ```editor``` check, and replaced ```mods``` check with an ```all``` check that includes ```Game``` and ```Editor```.

**Comment by Mailaender** [comment]
:+1: but someone on Windows needs to verify that this works.

**Comment by pchote** [comment]
Do we need all these individual stylecop-* options?  I think a single stylecop option that matches what travis does would be simpler.

**Comment by reaperrr** [comment]
> Do we need all these individual stylecop-* options? I think a single stylecop option that matches what travis does would be simpler.

I guess you're right. Originally I made this some days ago when Game and Mods.RA had not yet been stylecopped and would have created tons of noise, but by now a single check for all files is probably the better approach. Will update it later.

**Comment by reaperrr** [comment]
Updated.

**Comment by penev92** [comment]
Looks good. :+1: 
Can we make the messages colorful, though? :D  That would make it much easier to read.

File: Makefile [hunk 1]
```diff
@@ -9,6 +9,9 @@
 # to check the official mods for erroneous yaml files, run:
 #   make test
 #
+# to check the official mod dlls for StyleCop violations, run:
```
Comment (ScottNZ) [inline Makefile]: I think this should say "source code" instead of "official mod dlls"

**Pull request merged**